source/1_engine/rex_engine/src/diagnostics/win/win_stacktrace.cpp

```diff
@@ -0,0 +1,130 @@
+#include "rex_engine/diagnostics/win/win_stacktrace.h"
```

Comment:
This file could use a lot more documentation, as this is very windows specific and not everybody will be able to read this easily if bugs occur.

Reply:
it's in a "win" folder and has a "win" prefix, of course it's windows specific

Reply:
Yes so? 
I only asked to write documentation to the code.